.github/workflows/flutter-test.yml

```
@@ -10,7 +10,7 @@ jobs:
     name: Flutter Test on Linux
     permissions:
       contents: read
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest, macos-latest]
```

actionlint (1.7.7)

label "macos-latest" conflicts with label "ubuntu-latest" defined at line:13,col:15. note: to run your job on each workers, use matrix

(runner-label)